frontend/src/scenes/surveys/SurveyCustomization.tsx

```diff
@@ -74,12 +86,31 @@ export function Customization({
                     {validationErrors?.borderColor && <LemonField.Error error={validationErrors?.borderColor} />}
                 </LemonField.Pure>
                 <>
-                    <LemonField.Pure className="mt-2" label="Position">
+                    <LemonField.Pure
+                        className="mt-2"
+                        label="Position"
+                        info={
+                            isWidgetSurveyWithSelectorWidgetType
+                                ? 'The "next to trigger" option requires posthog.js version TODO_BEFORE_MERGE or higher.'
+                                : undefined
+                        }
+                    >
                         <div className="flex gap-1">
-                            {['left', 'center', 'right'].map((position) => {
+                            {Object.values(SurveyPosition).map((position) => {
+                                if (
+                                    position === SurveyPosition.NextToTrigger &&
+                                    !isWidgetSurveyWithSelectorWidgetType
+                                ) {
+                                    return null
+                                }
                                 return (
                                     <LemonButton
                                         key={position}
+                                        tooltip={
+                                            position === SurveyPosition.NextToTrigger
+                                                ? 'This option is only available for widget surveys. The survey will be displayed next to the trigger button.'
```

Comment:
i think widget should not be used here since the survey presentation name is feedback button right?

Reply:
yep, if there's any more UI labels with 'widget' i'd just replace it.
if our case survey type is widget (looking at the code), that's fine, we can just add a comment for it